plugins/autocertkit

```diff
@@ -1632,9 +1632,18 @@ def _get_local_device_bridge(session, device):
             log.debug("%s is a bridge." % device)
             return device
 
-    raise Exception("Error: Cannot determine bridge name of given device name %s." % device)
+    log.debug("Cannot determine bridge of given device %s." % device)
 
-    return devname
+    if device.startswith('xenbr'):
+        log.debug("Try using device name %s." % device)
+        return device
+
+    elif device.startswith('eth'):
```

Comment:
Initially this codes were just changing eth to xenbr if it starts with eth, this can cause issue due to change in XS 7.0. Newly created network can have different name than xenbrX from XS 7.0

See #115 for detail.

As we can assume all ethX has corresponding xenbrX, I think it is safe enough. Besides, as we cannot check this with xenapi, we need to use another local call to confirm (ifconfig and parsing), which I suppose waiste of resource.

I will update description to make it a bit clearer.